src/Tes.RunnerCLI/scripts/rootHash.sh

```diff
@@ -0,0 +1,21 @@
+﻿#!/bin/bash
```

Comment:
this file should probably be named "roothash.sh", since in Linux, lower case file names are canonical and it's case sensitive